### PR TITLE
Add can-connect-ndjson and can-ndjson-stream docs and tests

### PR DESCRIPTION
**Pull request by bgando**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,12 +39,14 @@
     "github": "https://github.com/canjs/canjs"
   },
   "dependencies": {
+    "bit-docs-generate-html": "^0.6.0",
```
Comment (nlundquist) [inline package.json]: this is already listed in the bit-docs dependencies so this wouldn't need to be here

File: package.json [hunk 2]
```diff
@@ -95,7 +98,7 @@
     "can-zone": "0.6.7"
   },
   "devDependencies": {
-    "bit-docs": "^0.0.7",
+    "bit-docs": "0.0.7",
```
Comment (chasenlehara) [inline package.json]: Why was this changed?

**Comment by bgando** [comment]
I am still experiencing these errors, but I do not believe it is related to can-connect-ndjson or can-ndjson-stream.
image
